### PR TITLE
Adding apply link to the Outreach position description

### PR DESCRIPTION
**Pull request by pixelwake**
This link will route to the Lever application for the Director of Outreach. This position will be the pilot to test before making the full cut over.

Adding the link to the application to each specific role to prepare for the cut over to the Lever hosted application.

To #189

### CODE REVIEW

File: _positions/director-of-outreach.md [hunk 1]
```diff
@@ -29,3 +29,5 @@ You’re a broad and strategic thinker who can see the interconnectivity between
 - Define team and individual performance metrics and expected business results through OKRs and regular calibration of team goals with individual goals.
 
 
+##[Apply to join 18F](https://jobs.lever.co/18f/1e419ac8-737d-4edb-b5ab-f186775246f1/apply)
```
Comment (fureigh) [inline _positions/director-of-outreach.md]: As this links to an application for one specific position, how about "Apply to be 18F's Director of Outreach" or "Apply to be our Director of Outreach"?

**Comment by fureigh** [comment]
One suggested change; otherwise looks :+1:

File: _positions/director-of-outreach.md (hunk 1 above)
Comment (pixelwake) [inline _positions/director-of-outreach.md]: Sure, but maybe for consistency we could say "Apply to the Director of Outreach role", that way we can keep the same format for something like software dev, "Apply to the Software Developer role".

Comment (fureigh) [inline _positions/director-of-outreach.md]: Sure, that sounds good. (I kind of liked the additional context that the singular vs. plural adds, as it tells the applicant whether there will be other people with the same role or whether they'll be the only one, but the position/role description itself should adequately cover that information.)

**Comment by fureigh** [comment]
Merging!

**Pull request merged**